rhea/build/boards/altera/_de1_SoC.py

```diff
@@ -0,0 +1,229 @@
+#
```

Comment:
No, I basically finished add the reset of the pins for the de0nano_soc, as per the todo comment in the file.
And made a mirror image file for the de1_soc